test/unittests/util/test_format_en.py

```diff
@@ -0,0 +1,370 @@
+						 "hundred and eighty nine billion, one hundred "
+						 "and twenty three million, four hundred and "
+						 "fifty six thousand, seven hundred and eighty nine")
+        self.assertEqual(pronounce_number(1), "one")
+        self.assertEqual(pronounce_number(10), "ten")
+        self.assertEqual(pronounce_number(15), "fifteen")
+        self.assertEqual(pronounce_number(20), "twenty")
+        self.assertEqual(pronounce_number(27), "twenty seven")
+        self.assertEqual(pronounce_number(30), "thirty")
+        self.assertEqual(pronounce_number(33), "thirty three")
+
+        self.assertEqual(pronounce_number(71), "seventy one")
+        self.assertEqual(pronounce_number(80), "eighty")
+        self.assertEqual(pronounce_number(74), "seventy four")
+        self.assertEqual(pronounce_number(79), "seventy nine")
+        self.assertEqual(pronounce_number(91), "ninety one")
+        self.assertEqual(pronounce_number(97), "ninety seven")
+        self.assertEqual(pronounce_number(300), "three hundred")
+
+    def test_convert_negative_int_de(self):
+        self.assertEqual(pronounce_number(-1), "minus one")
+        self.assertEqual(pronounce_number(-10), "minus ten")
+        self.assertEqual(pronounce_number(-15), "minus fifteen")
+        self.assertEqual(pronounce_number(-20), "minus twenty")
+        self.assertEqual(pronounce_number(-27), "minus twenty seven")
+        self.assertEqual(pronounce_number(-30), "minus thirty")
+        self.assertEqual(pronounce_number(-33), "minus thirty three")
+
+    def test_convert_decimals_de(self):
+        self.assertEqual(pronounce_number(21.234),
+                         "twenty one point two three")
+        self.assertEqual(pronounce_number(21.234, places=1),
+                         "twenty one point two")
+        self.assertEqual(pronounce_number(21.234, places=0),
+                         "twenty one")
+        self.assertEqual(pronounce_number(21.234, places=3),
+                         "twenty one point two three four")
+        self.assertEqual(pronounce_number(21.234, places=4),
+                         "twenty one point two three four")
+        self.assertEqual(pronounce_number(21.234, places=5),
+                         "twenty one point two three four")
+        self.assertEqual(pronounce_number(-21.234),
+                         "minus twenty one point two three")
+        self.assertEqual(pronounce_number(-21.234, places=1),
+                         "minus twenty one point two")
+        self.assertEqual(pronounce_number(-21.234, places=0),
+                         "minus twenty one")
+        self.assertEqual(pronounce_number(-21.234, places=3),
+                         "minus twenty one point two three four")
+        self.assertEqual(pronounce_number(-21.234, places=4),
+                         "minus twenty one point two three four")
+        self.assertEqual(pronounce_number(-21.234, places=5),
+                         "minus twenty one point two three four")
```

Comment:
why does it add `null` in german  when the precision is higher than the precision of the given number, while in english it does not add `zero` and `zero zero` in the end?

Reply:
"minus twenty one" is a perfectly ordinary, colloquial way of phrasing it in English, but I should point out that (at least to an American) it comes across as extremely informal, as colloquial as it gets. It might be worth considering, "negative twenty one," which is the most common phrasing.

"Minus x," in my experience, is most frequently used when referring to weather. Two minus three is negative one, but a cold temperature would be, "minus one degree."

Reply:
thats not what I mean:
I mean why in german it is:
`minus einundzwanzig komma zwei drei vier null null`
and in english
`minus twenty one point two three four`
and not 
`minus twenty one point two three four zero zero`
when precision is set to 5